contents/node/nodejs.md

```diff
@@ -151,45 +172,44 @@ For example, inside your `package.json`, you might have
+This makes development a breeze -- if a specific functionality has a large development overhead, you could look to such modules to speed things up.
 
-Importing modules is simple - just use the `require()` function and provide the module identifier or the file path.
+Importing modules is simple - simply use the `require()` function and provide the module identifier or the file path.
 ```js
 const http = require('http'); // import a core module
 const Promise = require('bluebird'); // import a non-core module
 ```
 
-Node will first check if the module identifier passed to `require` is a core module or a relative path. If so, it will return the core module or the object which references the value of `module.exports` in the specified file path respectively. Otherwise, Node will attempt to load the module from the `node_modules` folder in the parent directory of the current module.
-
-### Benefit 8: Relatively Large Developer Community
+Node will first check if the module identifier passed to `require` is a core module or a relative path. If so, it will return the core module or the value of `module.exports` in the specified file path's code. Otherwise, Node will attempt to load the module from the `node_modules` folder in the parent directory of the current module.
 
-npm (Node Package Manager) is the largest ecosystem of open source libraries in the world. You can find a wide range of helpful and well-tested packages that can serve your needs on [npmjs](https://www.npmjs.com/).
+### Benefit 8: Support for Better Code Organization
 
-### Benefit 9: Support for Better Code Organization
+With NodeJS's module system, you can create separate modules in our codebase such that each is focused on a single functionality. This makes your code more maintainable and testable.
 
-We can also create separate files and modules in our codebase such that each module is focused on a single functionality. This makes your code more maintainable and testable.
-
-To export the `Parser` constructor in `parser.js`,
+For example, we could have a module for a parser. To export the `Parser` constructor in `parser.js`,
 ```js
 function Parser(options) {
-    this._options = options || {};
+  this._options = options || {};
 }
 
 Parser.prototype.parse = function (content) {
-  // insert code here
+  ...
 }
 module.exports = Parser; // override the exports object
 ```
 
-The Parser constructor will then be returned whenever the `require` function is used to include the `Parser` module. It can be used to create Parser objects in other files in the project, for example,
+Elsewhere, the Parser constructor would then be imported as such, allowing Parser objects to be created in other modules.
```

Comment:
I would replace "as such," with "thus"

Reply:
Ah I see, my bad.

Reply:
No worries, thanks for the comprehensive review! 🙇